### ISSUE TITLE
Sidebar Classes Customization Issue

### ISSUE THREAD

**Issue by Vac1911** [Laravel-Backpack/CRUD#2472]
# Bug report

Hey! I'm new to backpack and backstrap but I came across a very small issue.

### What I did

I was attempting to style a new project as demonstrated [here](http://backpackforlaravel.com/docs/4.0/base-how-to#inspired-by-github) by adding classes to my 'config\backpack\base.php' file, specifically the class ```sidebar-light``` to 'sidebar_classes'.

### What I expected to happen

The rendered sidebar element to have the class ```sidebar-light``` added to its class list and appear white.

### What happened

The renderd sidebar did receive the the class. However the elements styling was unchanged. In the default stylesheets ```packages/backpack/base/css/bundle.css```, I found that there was no css selector for ```.sidebar-light``` in the file. (It's also absent in ```packages/@digitallyhappy/backstrap/css/style.min.css```).

### What I've already tried to fix it

I don't believe it is possible to accomplish the same styling effect by adding classes to the sidebar. Adding ```bg-white``` to the sidebar doesn't cut it because ```.nav-link``` items still have white text. (```bg-white text-dark``` also doesn't work because the ```.nav-link``` style overwrites the cascading text-color attribute).

The problem was easily solved by creating a .sidebar-light selector in my app.scss file. However that solution is outside the scope of the docs page. Because I am unsure of how the developers would want this resolved; I created a report rather than forking and creating a pull request.

### Backpack, Laravel, PHP, DB version

_PHP VERSION:_
PHP 7.4.2
_LARAVEL VERSION:_
laravel/framework                     v6.16.0 
_BACKPACK VERSION:_
backpack/crud                         4.0.41
backpack/generators                   2.0.6

**Comment by tabacitu** [comment]
Hmmm you're right @Vac1911 , thanks for bringing this up. It looks like we _used to_ have a ```sidebar-light```, but we no longer do. We've turned that into ```sidebar-pills``` which leaves out this one use case - when you want a white background sidebar, but with normal buttons, not pills.

Until we fix this, please use ```sidebar bg-white sidebar-pills``` for your sidebar classes, you will get this, which is close enough I think:
![Screenshot 2020-02-21 at 08 03 55](https://user-images.githubusercontent.com/1032474/75008451-bce6a280-5480-11ea-84f0-e25ee387d6ae.png)

I've changed the docs to have the solution above too - don't want other people to come across this and go through the frustrating process you must have gone through 😄 Sorry for that.

----

I'll move the issue to the Backstrap repo, since that's where the problem originated. 

Notes for debugging this:
- on 22 Sep 2019 we did add [style for sidebar-light](https://github.com/DigitallyHappy/BackStrap/commit/b6413f6d9a0f0776007ea0b8c6c02c9402db4d67);
- back then _I think_ we wanted to toggle between ```sidebar-dark```, ```sidebar-light``` and ```sidebar-transparent```;
- _I think_ we decided to make the _kind_ of sidebar independent from the _color_; so instead of having the three options above, we'd have:
   - ```sidebar``` - needed for the overall style and functionality;
   - ```sidebar-pills``` - make the nav items look like smaller pills, instead of full-width menu elements;
   - ```bg-white```, ```bg-dark```, ```bg-secondary``` etc - sidebar background color;
- in hindsight, that might not have been the best decision; sure, it provides more customization options, but I don't think people really want their sidebar to be a different color than white/black/transparent;
- anyway, here's the [git history of ```_custom.scss```](https://github.com/DigitallyHappy/BackStrap/commits/master/src/scss/_custom.scss);
- anyway, the problem with the solution above is that CoreUI does NOT provide a way to have a white sidebar; 

Code that should fix the problem:
```css

.sidebar-light .nav-link.active,
.sidebar-light .nav-link:hover {
  color: white!important;
}

.sidebar-light a:not(:hover),
.sidebar-light a:not(:active),
.sidebar-light a:not(:focus),
.sidebar-light a:not(.open),
.sidebar-light a:not(.active),
.sidebar-light li {
  color: #495057!important;
}

.sidebar-light .nav-dropdown.open {
  background: #fbfbfb;
  color: #495057!important;
}

.sidebar-light .nav-link.active {
    background: #f6f6f6;
    color: #467fd0!important;
}

.sidebar-light .nav-link.active .nav-icon,
.sidebar-light .nav-link.active:hover .nav-icon {
    color: #467fd0!important;
}
```

Use ```sidebar sidebar-light bg-white``` to get a white sidebar.

**Comment by tabacitu** [comment]
@Vac1911 - I've just pushed the fix as Backstrap 0.2.23 - on new installs this should be fixed. This will also be fixed in Backpack 4.1 when we do an update of all assets, so I'm going to close this.

Thanks for bringing this up. Sorry we can't push a fix for you right away, but the workarounds above should be good enough for you project.

Cheers!

**Comment by Vac1911** [comment]
@tabacitu No worries! I had already found a semi jank solution for myself, lol :D
